torch/csrc/jit/codegen/cuda/compute_at.cpp

```diff
@@ -756,11 +765,12 @@ void ComputeAt::runPass() {
   // Back off on inlining the inner broadcast axes
   hoistInnermostBroadcast();
 
+  // Clear max producer position of consumers from fusion inputs.
+  updateInputProduceAts();
+
```

Comment:
Why is `updateInputProduceAts` moved before `updateSiblings`? Not sure if there's any ordering requirement.

Reply:
I wanted to make sure all changes were done before updating the siblings. Just seemed to make more sense to me like that.